compiler/noirc_evaluator/src/ssa/parser/into_ssa.rs

```diff
@@ -558,10 +557,6 @@ impl Translator {
         // before each print.
         ssa.normalize_ids();
 
-        for function in ssa.functions.values() {
-            validate_function(function);
-        }
```

Comment:
Why is this gone? (maybe it's already done somewhere else and this was redundant?)